.circleci/config.yml

```diff
@@ -244,7 +263,7 @@ commands:
             $nuspecFilePath = "./SDKs/PowerShell/$ModuleName/$ModuleName.nuspec"
             $BuildVersion = If ('<< parameters.source >>' -eq 'CodeArtifact')
             {
-              "$($Psd1.ModuleVersion)-Build$($env:CIRCLE_BUILD_NUM)DateTime$((Get-Date).ToString("yyyyMMddHHmm"))"
+              "$($Psd1.ModuleVersion).$($env:CIRCLE_BUILD_NUM)-$((Get-Date).ToString("yyyyMMddHHmm"))"
```

Comment:
PowerShellGet can install the latest version of a nuget module if the modules if the build number does not contain a text string. Within CodeArtifact I'm saving these modules as major.minor.patch.build-datetime.

These latest versions of these dev modules can be installed with: 
`
Install-PSResource -Credential $RepositoryCredentials -Repository CodeArtifact -Name JumpCloud.sdk.v1 -Prerelease
Install-PSResource -Credential $RepositoryCredentials -Repository CodeArtifact -Name JumpCloud.sdk.v2 -Prerelease
Install-PSResource -Credential $RepositoryCredentials -Repository CodeArtifact -Name JumpCloud.sdk.directoryinsights -Prerelease
`
![Screen Shot 2022-01-05 at 1 37 31 PM](https://user-images.githubusercontent.com/54448601/148285826-b3386247-2298-4ef7-b430-18eeef7d9b04.png)

Reply:
thats cool